presto-docker-image/README.md

```diff
@@ -26,3 +26,63 @@ across all worker nodes if desired. Additionally this has the added benefit of
 The default configuration uses `/data/presto` as the default for
 `node.data-dir`. Thus if using the default configuration and a mounted volume
 is desired for the data directory it should be mounted to `/data/presto`.
+
+### Quickstart
+The following steps will guide to quick test PrestoSQL using docker image.
+
+ 1. Run the PrestoSQL docker
+
+  To launch single machine prestosql instance for testing which will function as both a coordinator and worker, just run:
+
+  ```bash
+  docker run -p 8080:8080 prestosql/presto:latest
+  ```
+  
+  NOTE: You need to have [docker](https://docs.docker.com/install/) installed in the system
+
+  Wait for the message: `INFO	main	io.prestosql.server.PrestoServer	======== SERVER STARTED ========`
+
+  Now PrestoSQL server is running at `localhost:8080` or `YOUR-INSTANCE-IP:8080`
+ 
+ 2. Connect to PrestoSQL 
+ 
+  To connect to PrestoSQL server: 
+  
+  Download the PrestoSQL CLI (terminal-based interactive shell for running queries) using command: 
+  ```bash
+  wget -O presto-cli-312 https://repo1.maven.org/maven2/io/prestosql/presto-cli/312/presto-cli-312-executable.jar
```

Comment:
You're using hard-coded version number here, so we should be updating this per-release.... better to have something that doesn't require maintenance

i strongly feel the docker image that we recommend for demo purposes should have presto-cli preinstalled, and available on `$PATH` (perhaps in `/usr/local/bin`). I think postgresql, myql, mssql docker images all do so and I find this very convenient.

So, I suggest
1. we add the cli to the image
2. we just use `docker exec -it presto presto-cli`

Reply:
Yeah!, That can help

Reply:
So the docker image does contain the CLI already and it is in the path at `/usr/bin/presto`

Reply:
Okay Update :)

Reply:
I just tried and this works today (if you use `--name presto` when starting the container):
```
docker exec -it presto presto
```

Reply:
Yeah, I updated that 
https://github.com/prestosql/presto/pull/868/files#diff-67eef4356830ac58f22e2395cca9b6c6R54